cmd/kubelet/app/server.go

```diff
@@ -647,31 +648,34 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 		}
 
 		var reservedSystemCPUs cpuset.CPUSet
+		useEmptyCPUSet := true
```

Comment:
you can avoid using this variable, just define it from the beginning 
`reservedSystemCPUs := cpuset.NewCPUSet()`

Reply:
ack